### PR TITLE
feat(Guides): Update qBittorrent Themes page

### PR DESCRIPTION
**Pull request by yammes08**
# Pull Request

## Purpose

Update qBittorrent Themes page

## Approach

- Rename Themes to Themes and UI replacements
- Add VueTorrent UI replacement

## Open Questions and Pre-Merge TODOs

None

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by yammes08** [comment]
Resolves #2029

File: docs/Downloaders/qBittorrent/Tips/Themes-and-UI-replacements.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-| ------------------------------------------------------------------------------------------------------------------------------- | ----- | --------------- |
-| [Honeywell](https://github.com/stacksmash76/qbt-honeywell){:target="\_blank" rel="noopener noreferrer"}                         | Dark  | v4.3.1 - v4.4.5 |
-| [Nightwalker Theme (CallMeBruce fork)](https://github.com/CallMeBruce/nightwalker){:target="\_blank" rel="noopener noreferrer"} | Dark  | v4.5            |
-| [World of Quinoa](https://github.com/gl0ryus/woq){:target="\_blank" rel="noopener noreferrer"}                                  | Dark  | v4.3.9, v4.4.5  |
+| Theme                                                                                                                           | Type           | Style      | Compatibility   |
+| ------------------------------------------------------------------------------------------------------------------------------- | -------------- | ---------- | --------------- |
+| [Honeywell](https://github.com/stacksmash76/qbt-honeywell){:target="\_blank" rel="noopener noreferrer"}                         | Theme          | Dark       | v4.3.1 - v4.4.5 |
```
Comment (TRaSH-) [inline docs/Downloaders/qBittorrent/Tips/Themes-and-UI-replacements.md:7]: This URL isn't valid

**Review by TRaSH- — CHANGES_REQUESTED** [review]

File: docs/Downloaders/qBittorrent/Tips/Themes-and-UI-replacements.md (hunk 1 above)
Comment (TRaSH-) [inline docs/Downloaders/qBittorrent/Tips/Themes-and-UI-replacements.md]: THis URL isn't valid anymore, and I can't find the replacement URL

Comment (yammes08) [reply docs/Downloaders/qBittorrent/Tips/Themes-and-UI-replacements.md]: Looks like the repo has been deleted - I've removed it.

**Review by TRaSH- — APPROVED** [review]

**Pull request merged**